### PR TITLE
Feat/neutrino

### PR DESCRIPTION
**Pull request by chrisguida**

### CODE REVIEW

File: assets/compat/config_spec.yaml [hunk 1]
```diff
@@ -115,21 +115,23 @@ bitcoind:
...
   description: |
     The Bitcoin Core node to connect to:
+      - none: Use the light bitcoin backend built into LND, Neutrino.
       - internal: The Bitcoin Core and Proxy services installed to your Embassy
-      - external: An unpruned Bitcoin Core node running on a different device
   tag:
     id: type
     name: Type
     variant-names:
+      none: None (Built-in LND Neutrino)
```
Comment (MattDHill) [inline assets/compat/config_spec.yaml:124]: I think this should be last in the list

File: assets/compat/config_spec.yaml [hunk 2]
```diff
@@ -115,21 +115,23 @@ bitcoind:
...
       internal: Internal (Bitcoin Core)
```
Comment (MattDHill) [inline assets/compat/config_spec.yaml:125]: No need for "internal". It's none, core, or proxy

File: assets/compat/config_spec.yaml [hunk 3]
```diff
@@ -115,21 +115,23 @@ bitcoind:
...
       internal-proxy: Internal (Bitcoin Proxy)
-      external: External
     description: |
       The Bitcoin Core node to connect to:
-        - internal: The Bitcoin Core and Proxy services installed to your Embassy
-        - external: An unpruned Bitcoin Core node running on a different device
+        - none: Use the light bitcoin backend built into LND, Neutrino. If using Neutrino, please switch to using Bitcoin Core as soon as possible. Neutrino uses the BIP157/8 light client protocol, which has security risks.
```
Comment (MattDHill) [inline assets/compat/config_spec.yaml:129]: "Please switch to using Bitcoin Core as soon as possible." is odd given this is an explicit, opt-in selection. Maybe "It is recommended to run LND against Bitcoin. Neutrino uses..."

File: assets/compat/config_spec.yaml (hunk 2 above)
Comment (ProofOfKeags) [reply assets/compat/config_spec.yaml:125]: I'd recommend against changing the enum variants, but we can change the title renders for sure.

**Review by ProofOfKeags — APPROVED** [review]

**Review by ProofOfKeags — APPROVED** [review]
SNED

File: assets/compat/config_spec.yaml (hunk 1 above)
Comment (chrisguida) [reply assets/compat/config_spec.yaml:124]: Interestingly, that's how it already appears. Was actually gonna file a bug about this haha

Screen Shot 2022-06-29 at 5 22 15 PM

File: assets/compat/config_spec.yaml (hunk 2 above)
Comment (chrisguida) [reply assets/compat/config_spec.yaml:125]: will change the copy once we go to master, this PR doesn't have the JS changes

Comment (chrisguida) [reply assets/compat/config_spec.yaml:125]: Copy change made here: https://github.com/Start9Labs/lnd-wrapper/commit/79e90b8f144d553ac5be90aec566140c6c1940de

**Pull request merged**